server/src/test/java/org/elasticsearch/common/time/FormatNamesTests.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.common.time;
+
+import junit.framework.TestCase;
+
+public class FormatNamesTests extends TestCase {
+
+    public void testDeprecation(){
+
```

Comment:
missing test?

Reply:
will remove this. I added deprecation test to https://github.com/elastic/elasticsearch/pull/58909/files#diff-198e1ff52598409599e5a427493f0227R386